process-phoenix/src/main/java/com/jakewharton/processphoenix/ProcessPhoenix.java

```diff
@@ -59,7 +59,7 @@ public static void triggerRebirth(Context context, Intent nextIntent) {
     intent.addFlags(FLAG_ACTIVITY_NEW_TASK); // In case we are called with non-Activity context.
     intent.putExtra(KEY_RESTART_INTENT, nextIntent);
     context.startActivity(intent);
-
+    ((Activity) context).finish();
```

Comment:
I removed the instance of check because @JakeWharton asked me to :)
"Just call finish() here. We know we're an activity." 
I knew crash would happen if someone would pass applicationContext but I simply thought I might be missing something.
I was even thinking of suggesting that we pass an activity as parameter instead of context, if we know we are an activity.